cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -366,6 +386,14 @@
 (decl xmm0 () WritableReg)
 (extern constructor xmm0 xmm0)
 
+;;;; Helpers for determining register class of value type ;;;;;;;;;;;;;;;;;;;;;;
+
+(decl is_reg_in_xmm (Type) Type)
+(extern extractor is_reg_in_xmm is_reg_in_xmm)
+
+(decl is_reg_in_gpr (Type) Type)
+(extern extractor is_reg_in_gpr is_reg_in_gpr)
```

Comment:
The `is_reg` prefix makes me think that these will extract from `Reg`s but they actually will extract from `Type`s. Can we rename these to `is_{xmm,gpr}_type` instead perhaps?

Reply:
Alternatively, we could have them operate on `Reg`s and keep the current names. Then we would just have to adjust the call sites appropriately, which I think would be possible for all call sites since they also have `Reg`s available.